ci-operator/step-registry/ipi/conf/gcp/osdisk/encryption-key/ipi-conf-gcp-osdisk-encryption-key-commands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ "${CONTROL_PLANE_OS_DISK_CUSTOM_ENCRYPTION}" != "yes" ]] && [[ "${COMPUTE_OS_DISK_CUSTOM_ENCRYPTION}" != "yes" ]]; then
```

Comment:
When calling the step in some chain/workflow, that means users want to test the feature on purpose, so we do not need the two ENV vars and the condition.